tools/apiview/parsers/csharp-api-parser/CSharpAPIParser/Program.cs

```diff
@@ -154,6 +151,40 @@ static async Task HandlePackageFileParsing(Stream stream, FileInfo packageFilePa
         }
     }
 
+    static async Task CreateOutPutFile(string outputPath, string outputFileNamePrefix, CodeFile apiViewFile)
+    {
+        var jsonTokenFilePath = Path.Combine(outputPath, $"{outputFileNamePrefix}.json");
+        await using FileStream fileStream = new(jsonTokenFilePath, FileMode.Create, FileAccess.Write);
+        await apiViewFile.SerializeAsync(fileStream);
+        Console.WriteLine($"TokenCodeFile File {jsonTokenFilePath} Generated Successfully.");
+        Console.WriteLine();
+    }
+
+    /*** Creates dummy API review file to support meta packages.*/
+    static CodeFile CreateDummyCodeFile(string originalName, string text)
+    {
+        var packageName = Path.GetFileNameWithoutExtension(originalName);
+        var packageNameMatch = _packageNameParser.Match(packageName);
+        var packageVersion = "";
+        if (packageNameMatch.Success)
+        {
+            packageName = packageNameMatch.Groups[1].Value;
+            packageVersion = $"{packageNameMatch.Groups[2].Value}";
```

Comment:
Does this assume package names will only be two parts? (like `Azure.Core` but never something like `Azure.Data.Tables`)

Reply:
Nope. regex parses something like `Azure.Storage.Blobs`.`10.12.0` or `Azure.Core`.`10.12.0`. REGEX format creates two groups.